examples/ProducerConsumerInstrument/property/ProducerConsumerInstrument.prop

```diff
@@ -0,0 +1,7 @@
+package property
+
+//Equality
```

Comment:
You probably want to have the same property name as the name of the function.
Although the property name must start with a capital letter.
So, change the property name to be EqualToZero

Reply:
I meant changing the name in the comment to EqualToZero from Equality. That is the name of the property.
equalToZero is the name of the function.